task/monorail-server/src/bsp/sidecar_1.rs

```diff
@@ -135,32 +135,12 @@ pub use map::PORT_MAP;
 /// which owns the ethernet peripheral containing the MDIO block.
 pub struct NetPhyRw<'a>(&'a mut task_net_api::Net);
 impl<'a> PhyRw for NetPhyRw<'a> {
-    #[inline(always)]
-    fn read_raw<T: From<u16>>(
-        &self,
-        port: u8,
-        reg: PhyRegisterAddress<T>,
-    ) -> Result<T, VscError> {
-        self.0
-            .smi_read(port, reg.addr)
-            .map(|r| r.into())
-            .map_err(|e| e.into())
+    fn read_raw(&self, port: u8, reg: u8) -> Result<u16, VscError> {
```

Comment:
Should this be annotated with `#[inline(never)]`, per the comment on the trait?

Reply:
So, in this case, the answer is no. This is just passing off to `smi_read` and should ideally be inlined.